tests/test_cli.py

```diff
@@ -285,6 +285,26 @@ def test_run_with_user_global_config_file(self):
             cli.run((os.path.join(self.wd, 'a.yaml'), ))
         self.assertEqual(ctx.returncode, 1)
 
+    def test_run_with_user_yamllintrc_config_file(self):
+        config = os.path.join(self.wd, 'fake-local-config')
+        self.addCleanup(os.remove, config)
```

Comment:
In other tests we use:
https://github.com/adrienverge/yamllint/blob/a54cbce1b62a37b76eb07479a16dd761d476990c/tests/test_config.py#L216

Maybe it would be better to stay consistent?

Reply:
100%